http/rest-client-reactive/src/main/java/io/quarkus/ts/http/restclient/reactive/fault/tolerance/RequestIdClientRequestFilter.java

```diff
@@ -22,6 +24,7 @@ public RequestIdClientRequestFilter(RequestIdManager requestIdManager) {
 
     @Override
     public void filter(ClientRequestContext requestContext) {
+        ContextLocals.get("UUID", null);
```

Comment:
When I look at this line "like first time seeing it", I have no idea what is a meaning of it.
Please add a comment why is this line here, what it changes and what it proves/verifies/ensures.